ranger-http/src/main/java/com/flipkart/ranger/http/model/ServiceDataSourceResponse.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015 Flipkart Internet Pvt. Ltd.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.flipkart.ranger.http.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.flipkart.ranger.core.model.Service;
+import lombok.*;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Why is this needed?

Reply:
In case any ranger server screws up the response object and misses a few fields. In the absence of it, exception will get handled though.